# Digital Inputs Values [1:8]
bool din_1
bool din_2
bool din_3
bool din_4
bool din_5
bool din_6
bool din_7
bool din_8